input/CIBLenz__353.yml

```diff
@@ -1,4 +1,5 @@
 mapper_class: MapperCIBLenz
+band: 353
```

Comment:
did we miss this?

Reply:
It wasn't needed because you specified the files to read. I added this to use it to create the map file names